packages/api/src/modules/file/resolvers.ts

```diff
@@ -0,0 +1,47 @@
+import { FileUpload } from 'graphql-upload';
+
+import { model as File } from '.';
+import { model as Project } from '../project';
+import { MAX_ALLOWED_FILE_SIZE_IN_BYTES } from './constants';
+
+interface ICreateFileArgs {
+  file: FileUpload;
+  projectId: string;
+  sourceLanguage: string;
+}
+
+async function createFile(parent, args: ICreateFileArgs, context) {
+  const { createReadStream, filename } = await args.file;
+
+  if (context.contentLength > MAX_ALLOWED_FILE_SIZE_IN_BYTES) {
+    throw new Error('File size exceeded, limit is 5MB.');
+  }
+
+  // const stream = createReadStream();
+  // stream.on('data', (chunk) => console.log(chunk.toString()));
+
+  const project = await Project.findOne({ _id: args.projectId });
+
+  if (!project) {
+    throw new Error('The provided project does not exist.');
```

Comment:
Sugiro substituir por `throw new ApolloError('The provided project does not exist.', 'PROJECT_NOT_FOUND');`